packages/acot-plugin-wcag/README.md

```diff
@@ -35,12 +35,11 @@ You can also enable all the recommended rules for our plugin. Add `plugin:@acot/
 
 <!-- acot-rules:start -->
 
-| Name                                                                                  | Description                                                     | :heavy_check_mark: |
-| :------------------------------------------------------------------------------------ | :-------------------------------------------------------------- | :----------------- |
-| [`@acot/wcag/button-has-name`](./docs/rules/button-has-name.md)                       | Ensures buttons have discernible text                           | :heavy_check_mark: |
-| [`@acot/wcag/dialog-open-and-close`](./docs/rules/dialog-open-and-close.md)           | Focus should work as expected when opening and closing dialogs. |                    |
-| [`@acot/wcag/interactive-supports-focus`](./docs/rules/interactive-supports-focus.md) | TBA                                                             | :heavy_check_mark: |
-| [`@acot/wcag/table-has-valid-role`](./docs/rules/table-has-valid-role.md)             | confirm table has table role                                    | :heavy_check_mark: |
```

Comment:
two rules, table-has-valid-role and dialog-open-and-close were removed by docgen command. Is it OK?

Reply:
Year, it's OK, I just forgot to delete it :ok_hand: